qds_sdk/cluster.py

```diff
@@ -274,6 +274,12 @@ def _parse_create_update(cls, args, action):
                                     help="location for ssh key to use to" +
                                          " login to the instance")
 
+        security_group.add_argument("--persistent-security-groups",
+                                    dest="persistent_security_groups",
+                                    help="a security group to associate with each" +
+                                         " node of the cluster. Typically used" +
+                                         " to provide access to external hosts")
+
```

Comment:
The help says `a security group...` but the argument name is `...groups`. Which one is correct? If the plural one is correct - let's also provide example usage in the help message.